java/src/jmri/jmrit/symbolicprog/SplitVariableValue.java

```diff
@@ -759,42 +761,41 @@ public void propertyChange(java.beans.PropertyChangeEvent e) {
                         }
                     }
                 }
-            } else {  // writing CVs
+            } else  if (_progState <= WRITING_FIRST) {  // writing CVs
```

Comment:
Static analysis fails with:
`Medium: Useless condition: it's known that this._progState <= -1 at this point [jmri.jmrit.symbolicprog.SplitVariableValue] At SplitVariableValue.java:[line 764] UC_USELESS_CONDITION`

I think you can solve that by changing the comparision from `<=` to `==`.

Reply:
@danielb987 Dave heap owns that class. I am not opposed to making a change to it myself, but I'd certainly like to keep my butchering to the SplitEnumVariableValue class if possible.

@dheap Is this something I should address?

Reply:
@TCSDCC-Jordan 
You added `if (_progState <= WRITING_FIRST)` to the `else` statement. That's causing this static analysis error. If you remove `if (_progState <= WRITING_FIRST)`, the problem will go away.

Reply:
Or maybe @dheap added that code. In either way, this needs to be addressed to get CI pass.

Reply:
@TCSDCC-Jordan @danielb987 The code I added was all for debugging purposes, not functional changes. I'm not sure why I changed this nor whether it indicates a functional coding error. Since I'm not in a position to access my dev machines at present, please make the suggested change and I'll flag it for checking later.

Reply:
No problem. I just made that change and pushed it up. Thanks!